### PR TITLE
Update deduping code for warn message.  Ignore x-amzn-trace-id.

### PR DESCRIPTION
**Pull request by labkey-matthewb**
#### Rationale
Reduce noise in server logs.

#### Related Pull Requests
-

### CODE REVIEW

File: api/src/org/labkey/api/security/AuthenticatedRequest.java [hunk 1]
```diff
@@ -465,7 +465,7 @@ HttpSession guestSession()
     }
 
     // helper to avoid filling the _log
-    private static final Cache<String, String> logMessages = CacheManager.getCache(100, TimeUnit.MINUTES.toMillis(1), "Guest session messages");
+    private static final Cache<String, String> logMessages = CacheManager.getCache(100, TimeUnit.MINUTES.toMillis(5), "Guest session messages");
 
     private static void _logGuestSession(String ip, String msg)
```
Comment (labkey-jeckels) [inline api/src/org/labkey/api/security/AuthenticatedRequest.java:470]: Is the IP in this scenario the IP of the client or the ALB?

**Review by labkey-jeckels — APPROVED** [review]
Seems like we don't need to change the log level with this change? It's hard for me follow the intent of the code in terms of logging. Maybe a comment or two to help explain?

File: api/src/org/labkey/api/security/AuthenticatedRequest.java (hunk 1 above)
Comment (labkey-matthewb) [reply api/src/org/labkey/api/security/AuthenticatedRequest.java:470]: Good question...

Comment (labkey-matthewb) [reply api/src/org/labkey/api/security/AuthenticatedRequest.java:470]: If there is an x-forwarded-for header we should probably use that.  In this case, it looks like there is no x-forwarded-for and the logged remote ips look like routable addresses.

**Pull request merged**